src/index.js

```diff
@@ -1 +1,43 @@
 import './style.css';
+import {
+  HomeApplication,
+  getAllCarsCount,
+} from './modules/home/home-app';
+import { showCommentModal } from './modules/comments/comments-app';
+import showReservationModal from './modules/reservations/reservation-app';
+import Icon from './images/logo.png';
+
+const homeApp = new HomeApplication();
+
+function registerHomeEvents() {
```

Comment:
- [ ] according to the requirements the Project should use ES6, which means you should avoid usage of the function keyword, please use arrow functions instead.